tests/test_ledger.re

```diff
@@ -108,14 +112,7 @@ describe("ledger", ({test, _}) => {
     expect_balance(c, t1, 0, t);
     expect_balance(c, t2, 5, t);
 
-    let t =
-      transfer(
-        ~source=a,
-        ~destination=c,
-        Amount.of_int(99),
-        make_ticket(~data=t1.data, ()),
-        t,
-      );
+    let t = transfer(~source=a, ~destination=c, Amount.of_int(99), t1, t);
```

Comment:
approving this ofc bc it's important but can you remind me why `99` is chosen here?

Reply:
it's just all the balance available of ticket t1